README.md

```diff
@@ -16,9 +18,44 @@ aims to apply the same technique for bootable host systems - using
 standard OCI/Docker containers as a transport and delivery format
 for base operating system updates.
 
-## More information
+## Generated container images
+
+At the moment the main image is `quay.io/centos-bootc/centos-bootc:stream9`
+
+## Building
+
+The build process is 95% just installing RPM content into a container
+image; there is some non-RPM config files etc. in this repository
+too.
+
+See:
+
+- [Containerfile.centos-stream9]
+- [Containerfile.centos-stream10]
+- [Containerfile.fedora-40]
+
+For example:
+
+`podman build --security-opt=label=disable --cap-add=all --device /dev/fuse -f Containerfile.centos-stream9 -t localhost/centos-bootc:stream9`
+
+NOTE: This will not work when using "podman machine" over the remote client
```

Comment:
What is blowing up on a MAC?

Reply:
Well I didn't dig deep into debugging it honestly.  On Linux at least I get a really weird error 

> error: Installing packages: Updating rpm-md repo 'baseos': metadata[process] already locked by /proc/self/exe(17)

I haven't even tried to debug but I suspect the problem has something to do with 9p (which has more bugs than virtiofs)...we need to convert podman-machine on Linux to use libvirt probably.